### PR TITLE
add ClickHouse to onprem database

### PR DESCRIPTION
**Pull request by Slach**
Signed-off-by: Slach <bloodjazman@gmail.com>

### CODE REVIEW

**Comment by mingrammer** [comment]
Please follow the contributing guideline: https://github.com/mingrammer/diagrams/blob/master/CONTRIBUTING.md

It doesn't seem to have been added via `autogen.sh`.

**Comment by Slach** [comment]
@mingrammer 
could you suggest, 
after `autogen.sh` run, many docs/nodes/*.md files were changed
should I add these changes too?

File: diagrams/onprem/database.py [hunk 1]
```diff
@@ -80,3 +84,4 @@ class Scylla(_Database):
 MSSQL = Mssql
 MySQL = Mysql
 PostgreSQL = Postgresql
+ClickHouse = Clickhouse
```
Comment (mingrammer) [inline diagrams/onprem/database.py]: Add a new line at EOF please.

**Comment by mingrammer** [comment]
Resolve the conflict please!

**Comment by Slach** [comment]
conflicts resolved @mingrammer please look at PR again

**Pull request merged**